src/google/protobuf/compiler/objectivec/objectivec_helpers.h

```diff
@@ -133,6 +133,22 @@ enum ObjectiveCType {
   OBJECTIVECTYPE_MESSAGE
 };
 
+template<class TDescriptor>
+string GetOptionalDeprecatedAttribute(const TDescriptor* descriptor, bool preSpace = true, bool postSpace = false) {
```

Comment:
Sounds reasonable. 
Should i again squash main commit and commit for this change, or 2 commits is ok?